src/main/kotlin/graphics/scenery/Arrow.kt

```diff
@@ -154,8 +153,8 @@ class Arrow(var vector: GLVector) : Node("Arrow"), HasGeometry {
 
     /**
```

Comment:
Codacy does not like comments over private functions, and I think also you can remove the comment -- the effects of the function are clear enough from its contents 👍

Reply:
OK, done.
Also, I've re-indented pieces of the code where tabs were used instead of `    ` (four spaces).